Todo.js

```diff
@@ -0,0 +1,43 @@
+import { acceptedTodo, deleteTodo } from './main'
+import { v4 as uuidv4 } from 'uuid'
+import $ from 'jquery'
+
+const todoList = $('.todo__list')
+
+export class Todo {
+  constructor(text) {
+    this.isDone = false
+    this.text = text
+  }
+
+  createNewTodo(text) {
+    const todo = $('<li>', {
+      class: 'todo__item',
+      text,
+      'data-id': uuidv4()
+    }).appendTo(todoList)
```

Comment:
no, class shouldn't know what to append it to.
It should be done outside the class